src/neuroglancer/sliceview/png/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# 2.0.24 (2021-06-10)
+FROM emscripten/emsdk@sha256:81ec54b7a096d28f24d906955dbf98ff336cca47658d980c243baa36f6484f9f
+
+RUN mkdir -p /usr/src/spng \
+    && curl -SL https://github.com/randy408/libspng/archive/refs/tags/v0.7.1.tar.gz \
+    | tar -xzC /usr/src/spng --strip-components=1
+
+RUN mkdir -p /usr/src/miniz \
+    && curl -SL https://github.com/richgel999/miniz/archive/refs/tags/2.2.0.tar.gz \
+    | tar -xzC /usr/src/miniz --strip-components=1
+
```

Comment:
Can you also validate the checksums using sha256sum?

Reply:
Whoops! Sorry I forgot. I'll do that now.

Reply:
I seem to be having some trouble getting it to exit. Am I doing this right?

ENV SPNG_0_7_1_SHA256 0726a4914ad7155028f3baa94027244d439cd2a2fbe8daf780c2150c4c951d8e

RUN mkdir -p /usr/src/spng \
    && curl -SL -o /usr/src/spng.tar.gz https://github.com/randy408/libspng/archive/refs/tags/v0.7.1.tar.gz 

RUN if [ $(echo "$(SPNG_0_7_1_SHA256) /usr/src/spng.tar.gz" | sha256sum --check --status) ]; then exit 1; fi;
RUN tar -xzC /usr/src/spng -f /usr/src/spng.tar.gz --strip-components=1 \
    && rm /usr/src/spng.tar.gz

Reply:
The `[` command is actually a shell command that treats its arguments as a condition.  That is useful for comparing strings or numbers, but in this case we are just testing the exit code of sha256sum so we don't need that.  Instead you can just use:

```dockerfile
RUN echo "${SPNG_0_7_1_SHA256} /usr/src/spng.tar.gz" | sha256sum --check --status
```
Note that you should also use `${...}` instead of `$(...)` for environment variable substitution.

Reply:
Thank you Jeremy! Fixed. My bash skills could use some work. I tried something like that, but it didn't work for me before. It's probably the curlies that made the difference.